tools/orca/src/models/model_basic.rs

```diff
@@ -43,6 +43,7 @@ impl ActorModel for ActorBasic {
         let (result, event) = match transition.action {
             TransitionAction::Login => model::login(client, person).await,
             TransitionAction::Logout => model::logout(client, person).await,
+            _ => Err(Error::InvalidState),
```

Comment:
File should be named basic.rs since we are in the dir "models".

Reply:
wopsies I'll fix it